source/cluster_manager/matcher.js

```diff
@@ -68,6 +68,33 @@ var webrtcMatcher = function () {
     };
 };
 
+var streamingMatcher = function () {
```

Comment:
is `streamingMatcher` same as `webrtcMatcher`?
If so, why not write `streamingMatcher = webrtcMatcher` instead of duplicating same code here?